ts/components/basic/Flex.tsx

```diff
@@ -1,12 +1,13 @@
 import styled from 'styled-components';
+import { HTMLDirection } from '../../util/i18n';
 
 export interface FlexProps {
   children?: any;
   className?: string;
   container?: boolean;
   dataTestId?: string;
   /****** Container Props ********/
-  flexDirection?: 'row' | 'column';
+  flexDirection?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
```

Comment:
I thought the rtl logic would take care of reversing the flex flow already?

Reply:
It does. But I didn't know it until after I added this. Thought it was worth keeping in case we ever needed it in future. Happy to remove if you want?